gramps/gen/plug/_pluginreg.py

```diff
@@ -1123,7 +1128,7 @@ def scan_dir(self, dir, uistate=None):
         extlen = -len(ext)
         pymod = re.compile(r"^(.*)\.py$")
 
-        for filename in os.listdir(dir):
+        for filename in filenames:
             name = os.path.split(filename)[1]
```

Comment:
Is this function called from anywhere else? This should be a no-op since the result of `os.walk` is _just_ the file names.